packages/integration-tests/projects/suite-web/tests/wallet/bitcoin-send-form.test.ts

```diff
@@ -50,6 +50,23 @@ describe('Send form for bitcoin', () => {
         // assert final state of form using screenshot
         cy.getTestElement('@wallet/send/outputs-and-options').matchImageSnapshot('bitcoin-send');
     });
+
+    it('add and remove output in send form, toggle form options, input data', () => {
```

Comment:
please use different message for the new `it()`, this one is duplicated

Also I can't see the new bitcoin-send-sats snapshot in the artifacts https://gitlab.com/satoshilabs/trezor/trezor-suite/-/jobs/2762910362/artifacts/browse/packages/integration-tests/projects/suite-web/snapshots/wallet/bitcoin-send-form.test.ts/ , not sure why

Reply:
done